### ISSUE TITLE
.hadolint.yml with docker image or warning suppression

### ISSUE THREAD

**Issue by keiranmraine** [hadolint/hadolint#635]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

Want to suppress a warning class via`.hadolint.yml`.  Doesn't appear to work via the docker image

### Actual behavior

Still shows `DL3008` warnings.

### Steps to reproduce the behavior

Used example from `README.md` to mount the config.

```bash
$ cat .hadolint.yml 
ignored:
  - DL3008
(venv) kr2@kr2-develop-20:pyssa$ docker run --rm -i -v $(pwd)/.hadolint.yml:/root/.config/hadolint.yaml hadolint/hadolint < Dockerfile | grep -c DL3008
5
```

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version` or
  `docker run --rm ghcr.io/hadolint/hadolint hadolint --version`:**

```bash
$ docker run --rm hadolint/hadolint hadolint --version
Haskell Dockerfile Linter v2.4.0-0-gf46ed85
```

**Additional environment details (OS, stack version, etc.)**

```bash
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.1 LTS
Release:	20.04
Codename:	focal
$ docker --version
Docker version 20.10.3, build 48d30b5

```

**Comment by lorenzo** [comment]
I wonder if this is related to https://github.com/hadolint/hadolint/pull/634

**Comment by keiranmraine** [comment]
> I wonder if this is related to #634

I didn't use relative paths as I know they are not compatible with volume mounts, `$(pwd)` is absolute.

**Comment by lorenzo** [comment]
I think what the other issue is saying is that `/root/.config/hadolint.yaml` should be `/.config/hadolint.yaml` instead

**Comment by maxnatt** [comment]
Yep, wrong config path in the container, as described in the #634.
